crates/holochain/tests/inline_zome_sketch.rs

```diff
@@ -0,0 +1,178 @@
+use futures::StreamExt;
+use hdk3::prelude::*;
+use holochain::{conductor::Conductor, core::ribosome::ZomeCallInvocation};
+use holochain_keystore::KeystoreSender;
+use holochain_state::test_utils::test_environments;
+use holochain_types::{
+    app::InstalledCell,
+    dna::{zome::inline_zome::InlineZome, DnaFile},
+};
+use holochain_zome_types::element::ElementEntry;
+use unwrap_to::unwrap_to;
+
+fn simple_crud_zome() -> InlineZome {
+    let entry_def = EntryDef::new(
+        "entry".into(),
+        Default::default(),
+        Default::default(),
+        Default::default(),
+        Default::default(),
+    );
+
+    InlineZome::new("", vec![entry_def.clone()])
+        .callback("create", move |api, ()| {
+            let entry_def_id: EntryDefId = entry_def.id.clone();
+            let entry = Entry::app(().try_into().unwrap()).unwrap();
+            let hash = api.create((entry_def_id, entry))?;
+            Ok(hash)
+        })
+        .callback("read", |api, hash: HeaderHash| {
+            api.get((hash.into(), GetOptions::default()))
+                .map_err(Into::into)
+        })
+}
```

Comment:
This is an example of creating an InlineZome with two callback functions and 1 EntryDef.